core/tests/protractor/accessibility.js

```diff
@@ -41,6 +61,364 @@ describe('screenreader and keyboard user accessibility features', function() {
         .getAttribute('id'));
   });
 
+  it('should test the navigation shortcut ctrl+0',
+    async function() {
+      await browser.get('get-started');
+      await waitFor.urlRedirection('http://localhost:9001/get-started');
```

Comment:
Why don't we use the constant `GET_STARTED_URL` here? Ditto for the other URLs below

Reply:
Done